Bio/Wise/psw.py

```diff
@@ -133,7 +143,7 @@ def align(pair,
           gap_start=None,
           gap_extension=None,
           *args, **keywds):
```

Comment:
This may be a matter of taste, but I would put all the arguments in one or two lines (not exceeding 79 characters).

Reply:
Largely personal taste, and we have no compelling reason to change this as part of this pull request.

(Separately I am becoming a fan of the code formatting tool ``black`` which would decide this kind of thing automatically)